<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,3,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="fonctionLabel">
        <property name="text">
         <string>Fonction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fonctionComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Borne inf</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="borneInfLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Borne sup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="borneSupLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="plotWidget" native="true"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>68</red>
            <green>112</green>
            <blue>173</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>68</red>
            <green>112</green>
            <blue>173</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>68</red>
            <green>112</green>
            <blue>173</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>35</red>
            <green>60</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true">color : #233c67;
</string>
      </property>
      <property name="title">
       <string>Intégration</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1,0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>nombre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="nombreSlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="gaucheSumRadioButton">
             <property name="text">
              <string>Somme à gauche</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="droiteSumRadioButton">
             <property name="text">
              <string>Somme à droite</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="calculerPushButton">
           <property name="text">
            <string>Calculer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,1">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Somme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sumLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Intégrale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="integraleLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="exporterAction"/>
    <addaction name="quitterAction"/>
   </widget>
   <widget class="QMenu" name="menuFonctions">
    <property name="title">
     <string>Fonctions</string>
    </property>
    <addaction name="listFonctionsAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFonctions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="exporterAction">
   <property name="text">
    <string>Exporter</string>
   </property>
  </action>
  <action name="quitterAction">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="listFonctionsAction">
   <property name="text">
    <string>Liste fonctions</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
